plutus-ledger/plutus-ledger.cabal

```diff
@@ -102,8 +104,11 @@ library
         plutus-tx -any,
         serialise -any,
         servant -any,
+        scientific -any,
         template-haskell -any,
         text -any,
+        time -any,
```

Comment:
Added the `time` package to have access to the `getPOSIXTime` function to implement `getCurrentSlot`.